security-jpa-quickstart/README.md

```diff
@@ -16,6 +16,8 @@ The database can be started using:
 
 Once the database is up you can start your Quarkus application.
 
+Note you do not need to start the database when running your application in dev mode or testing.
```

Comment:
```suggestion
Note you do not need to start the database when running your application in dev mode or testing. It will be started automatically as a Dev Service.
```